### DISCUSSION TITLE
YOLO11 resume traning

### DISCUSSION THREAD

**Discussion by SergioY66** [ultralytics/ultralytics#22064] [Q&A]
Hi there!
Im a new in computer vision. 
Im trying to solve one task using YOLO11.
I have some dataset and train a model woth 500 epoches. The results (mAP50-95 is not quite good). At the test resource I have absolute accuracy of 93%. The 7% is errors... This is too big for my task. I belive that 1000 or 1200 epoches will be enough. So I want to start new training process for 500 - 700 epoches with the same dataset and using early training model as a pretraining.  Is this porcess equal to training with 1000 - 1200 epoches? Thank you for advance.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @SergioY66, thanks for starting the discussion and for using YOLO11 🚀! This is an automated response to help you get started—an Ultralytics engineer will also jump in to assist you soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Reply by glenn-jocher** [reply]
Great question—yes: if you resume from your last checkpoint with `resume=True`, it’s effectively the same as continuing one long run to 1000–1200 epochs because the optimizer, LR scheduler, and EMA state are restored; starting a new run with `model=last.pt` but without `resume` is not equivalent as it resets those states. See the docs on resuming for details in [Resuming training](https://docs.ultralytics.com/modes/train/#resuming-interrupted-trainings).

bash
yolo train resume model=path/to/last.pt epochs=1200  # continue to a total of 1200 epochs

python
from ultralytics import YOLO
model = YOLO("path/to/last.pt")
model.train(resume=True, epochs=1200)

If validation has plateaued, more epochs alone may not close a 7% gap—consider checking labels, class balance, or trying a larger YOLO11 variant and tuning `imgsz`/augmentations; key knobs are summarized under [Train settings](https://docs.ultralytics.com/modes/train/#train-settings).
